spec/services/curator/digital_object_factory_service_spec.rb

```diff
@@ -171,7 +171,7 @@
           let(:json_names) { desc_json['name_roles'].map { |nr| nr['name'] } }
           let(:json_roles) { desc_json['name_roles'].map { |nr| nr['role'] } }
           it 'sets the correct number of names and roles' do
-            expect(name_roles.count).to eq 2
+            expect(name_roles.count).to be >= 2
```

Comment:
Would prefer to keep this an `eq` comparison with an exact number of expected results, just seems a little more reliable.

Reply:
Ok I actually added an additional `name_role` to the fixture that's being used in this example to ill just put `eq 3` in this case